Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/HOKSAMLAssertionBuilder.java

```diff
@@ -690,4 +707,17 @@ public static Evidence buildEvidence(String evAssertionID, DateTime issueInstant
     private static String createAssertionId() {
         return ID_PREFIX.concat(String.valueOf(UUID.randomUUID())).replaceAll("-", "");
     }
+
+    protected boolean isAuthDEvidenceConditionsDefaultValueEnabled() {
+        //if not provided or invalid return true else false
+        try {
+            String authDEvidenceConditionsDefaultValueEnabled = PropertyAccessor.getInstance().getProperty(NhincConstants.GATEWAY_PROPERTY_FILE, NhincConstants.ENABLE_AUTH_DEC_EVIDENCE_CONDITIONS_DEFAULT_VALUE);
+            if (authDEvidenceConditionsDefaultValueEnabled != null) {
+                return !authDEvidenceConditionsDefaultValueEnabled.equals(Boolean.FALSE.toString());
```

Comment:
Understand that it's logically right but the statement itself is confusing.  You're equating a double negative instead of just directly getting the value.